h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
Yes, I think it would be a good idea to remove this hard-coded way of returning the `expires_in` and calculate the amount of time until the token expires and return that. And for non-expiring tokens it should probably not include this key in the response (depending on what the RFC says).